src/_guides/libraries/library-tour.md

```diff
@@ -76,9 +75,7 @@ assert(double.parse('1.2e+2') == 120.0);
 {% endprettify %}
 
 For more information, see the API documentation for
-[int,]({{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-core/int-class.html)
-[double,]({{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-core/double-class.html) and
-[num.]({{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-core/num-class.html) Also see
+[int,][int] [double,][double] and [num.][num] Also see
```

Comment:
nit: `[num][num].` I think the period and the commas should be outside the link.

Reply:
Also, I believe you can just say `[num][]` in this case.

Reply:
Right, but those render in the HTML as external links, which have an arrow icon afterwards:
> <img width="630" alt="screen shot 2017-11-21 at 08 12 21" src="https://user-images.githubusercontent.com/4140793/33074326-bb9daf38-ce93-11e7-8e96-3b482582b662.png">

Kathy much prefers having the punctuation before the "external" icon.